docs/source/common/trainer.rst

```diff
@@ -266,13 +272,15 @@ accumulate_grad_batches
 Accumulates grads every k batches or as set up in the dict.
 Trainer also calls ``optimizer.step()`` for the last indivisible step number.
 
-.. testcode::
+.. testcode:: python
```

Comment:
we prefer `testcode` if possible

Reply:
I think I saw this usage in `optimizers.rst` so I wanted to follow the same but it's alright if we prefer just `testcode` In my next commit I reverted those. Thank you for letting me know!